packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -968,12 +967,14 @@ export class PipelineEditor extends React.Component<
                   pipelineJson,
                   this.widgetContext.path
                 );
-                this.setAndVerifyPipelineFlow(pipelineJson);
+                await this.setAndVerifyPipelineFlow(pipelineJson);
```

Comment:
this line doesn't build. You can't put an await inside a non-async function (which this `then` function is not)

Reply:
sorry, had the change locally to test but did after the commit...